### DISCUSSION TITLE
How to change rook-operator installation to helm without damaging the cluster

### DISCUSSION THREAD

**Discussion by ahmedTouati** [rook/rook#7957] [Q&A] [answered]
Hello, we've been using the cluster-on-pvc installation with rook-operator as a manifest for a while and now we want to change the rook-operator which is created by manifests to rook-operator helm chart and we want it to handle crd creation, so i annotated all the resources in common.yaml and crd.yaml with :
 **meta.helm.sh/release-name: rook-operator
 meta.helm.sh/release-namespace: rook-ceph**
and added this label:
**app.kubernetes.io/managed-by: Helm**

So i want to know if doing that will damage my cluster or not
Thanks,

**Comment by travisn** [comment, marked as answer]
@ahmedTouati Conceptually it would work. I'm not sure you would need to add those annotations to the CRDs. But in any case, do try it in a test cluster first and let us know if you run into any issues.

**Reply by ahmedTouati** [reply]
We've just tested it and it works fine we removed the old rook-operator and replaced it with rook-operator helm chart
